tooling/nargo_cli/build.rs

```diff
@@ -59,6 +59,12 @@ const IGNORED_BRILLIG_TESTS: [&str; 11] = [
     "is_unconstrained",
 ];
 
+/// Tests which aren't expected to work with the default inliner cases.
+const INLINER_MIN_OVERRIDES: [(&str, i64); 1] = [
+    // 0 works if PoseidonHasher::write is tagged as `inline_always`, otherwise 22.
+    ("eddsa", 0),
+];
+
```

Comment:
Can be removed now right?

Reply:
I don't think so, the effect of this is to not run this test with `i64::MIN`, which would still fail, because other functions also need to be inlined; they just happened to have negative costs, and they are inlined if we changed the default to 0.

Reply:
Wasn't that solved by inlining the poseidon hasher though?

Reply:
I'm afraid not. I printed the cost of inlining below, you can see the negative ones are inlined when we run it with `--inliner-aggressiveness 0`, nut not with `-Inf`. 

It just gave me the idea that I should find out what the minimum aggressiveness is to make the test pass, and it turned out to be 22, which meant `write` was the tipping point. Then I tried to see what happens if I inline `write`, but don't bother with `assert_gt` and others with low positive costs. It made the test pass, and I thought okay, at least `eddsa` won't fail if someone runs `nargo execute` on it. 

I haven't tried inlining random ones from the negative list to see if they alone would make it pass with `i64::MIN`, but ostensibly we can try now how low we can go before it fails again.

```
func=absorb cost=-5
func=add cost=-15
func=add cost=158
func=add cost=74
func=alpha cost=-1
func=apply_matrix cost=75
func=assert_gt cost=8
func=assert_gt_limbs cost=12
func=assert_max_bit_size cost=2
func=baby_jubjub cost=-23
func=baby_jubjub cost=14
func=bit_mul cost=-15
func=compute_decomposition cost=14
func=config cost=-16
func=contains cost=9
func=decompose cost=10
func=decompose_hint cost=-3
func=default cost=-2
func=eddsa_poseidon_verify cost=7995
func=eddsa_poseidon_verify cost=9857
func=eddsa_to_pub cost=-3
func=eddsa_verify cost=-7
func=eq cost=-9
func=field_less_than cost=-7
func=finish cost=-3
func=gt cost=-3
func=hash_1 cost=-2
func=hash_10 cost=-2
func=hash_11 cost=-2
func=hash_12 cost=-2
func=hash_13 cost=-2
func=hash_14 cost=-2
func=hash_15 cost=-2
func=hash_2 cost=-2
func=hash_3 cost=-2
func=hash_4 cost=-2
func=hash_5 cost=-2
func=hash_6 cost=-2
func=hash_7 cost=-2
func=hash_8 cost=-2
func=hash_9 cost=-2
func=into_affine cost=-7
func=into_group cost=-10
func=is_bn254 cost=-1
func=is_zero cost=-2
func=is_zero cost=4
func=lt cost=-3
func=lt cost=59
func=lt_fallback cost=-3
func=lte_hint cost=-3
func=main cost=-3
func=mixed_add cost=-14
func=mul cost=-15
func=mul cost=743
func=mul cost=747
func=new cost=-29
func=new cost=-31
func=new cost=-5
func=new cost=-8
func=perform_duplex cost=40
func=permute cost=-10
func=pow_32 cost=319
func=pow_32 cost=361
func=sigma cost=61
func=squeeze cost=-5
func=to_le_bytes cost=43
func=to_le_radix cost=-3
func=write cost=21
func=x5_10 cost=-2
func=x5_10_config cost=-8
func=x5_11 cost=-2
func=x5_11_config cost=-8
func=x5_12 cost=-2
func=x5_12_config cost=-8
func=x5_13 cost=-2
func=x5_13_config cost=-8
func=x5_14 cost=-2
func=x5_14_config cost=-8
func=x5_15 cost=-2
func=x5_15_config cost=-8
func=x5_16 cost=-2
func=x5_16_config cost=-8
func=x5_2 cost=-2
func=x5_2_config cost=-8
func=x5_3 cost=-2
func=x5_3_config cost=-8
func=x5_4 cost=-2
func=x5_4_config cost=-8
func=x5_5 cost=-2
func=x5_5_config cost=-8
func=x5_6 cost=-2
func=x5_6_config cost=-8
func=x5_7 cost=-2
func=x5_7_config cost=-8
func=x5_8 cost=-2
func=x5_8_config cost=-8
func=x5_9 cost=-2
func=x5_9_config cost=-8
func=zero cost=-2
func=zero cost=-5
```

Reply:
It already fails with `--inliner-aggressiveness -2`. -1 works.

Reply:
Nope, sorry I forgot. Thanks for the ping.